src/trans/algor/interpol_decomp_mod.F90

```diff
@@ -23,7 +23,7 @@ MODULE INTERPOL_DECOMP_MOD
 ! Author: Mats Hamrud
 
 
-USE PARKIND1, ONLY : JPRB, JPIM, JPRD, JPIB
+USE EC_PARKIND, ONLY : JPIM, JPRD, JPIB
```

Comment:
What's the purpose of `EC_PARKIND`?

Reply:
Excellent question. 
ec_parkind is part of fiat and does not contain JPRB so it shows this routine or module is precision independent and does not need to link to library parkind_sp or parkind_dp.